Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -109,30 +117,54 @@ $drop = CreateTestDrop $headers $dropjson $CltAccountUrl
+	Write-Output "To view detailed results navigate to Load Test | Load Test Manager in Visual Studio IDE, and open this run."
+
+	$resultsMDFolder = New-Item -ItemType Directory -Force -Path "$env:Temp\LoadTestResultSummary"	
+	$resultFilePattern = ("QuickPerfTestResults_{0}_{1}_*.md" -f $env:AGENT_ID, $env:SYSTEM_DEFINITIONID)
+	$excludeFilePattern = ("QuickPerfTestResults_{0}_{1}_{2}_*.md" -f $env:AGENT_ID, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID)
 	
-    Write-Output ("Run-id for this load test is {0} and its name is '{1}'." -f  $run.runNumber, $run.name)
-    Write-Output ("To view run details navigate to {0}" -f $webResultsUrl)
-    Write-Output "To view detailed results navigate to Load Test | Load Test Manager in Visual Studio IDE, and open this run."
+	if($avgResponseTimeThreshold)
+	{
+		$avgResponseTimeThresholdValidationResult = ValidateAvgResponseTimeThreshold $CltAccountUrl $headers $run.id $avgResponseTimeThreshold 
+		if($avgResponseTimeThresholdValidationResult -eq $false)
+		{
+			Write-Output "The Avg.Response Time for the run is greater than the threshold value of $avgResponseTimeThreshold specified for the run "
+			Write-Output "To view detailed results navigate to Load Test | Load Test Manager in Visual Studio IDE, and open this run."
+			Write-Error "Load test task is marked as failed, as there were threshold violations for the Avg. Response Time"
+		}
+	}
 
-    $resultsMDFolder = New-Item -ItemType Directory -Force -Path "$env:Temp\LoadTestResultSummary"	
-    $resultFilePattern = ("QuickPerfTestResults_{0}_{1}_*.md" -f $env:AGENT_ID, $env:SYSTEM_DEFINITIONID)
-    $excludeFilePattern = ("QuickPerfTestResults_{0}_{1}_{2}_*.md" -f $env:AGENT_ID, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID)   
-    Remove-Item $resultsMDFolder\$resultFilePattern -Exclude $excludeFilePattern -Force	
-    $summaryFile =  ("{0}\QuickPerfTestResults_{1}_{2}_{3}_{4}.md" -f $resultsMDFolder, $env:AGENT_ID, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID, $run.id)	
-    $summary = ('[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUrl ,$run.name)
+	Remove-Item $resultsMDFolder\$resultFilePattern -Exclude $excludeFilePattern -Force	
+	$summaryFile =  ("{0}\QuickPerfTestResults_{1}_{2}_{3}_{4}.md" -f $resultsMDFolder, $env:AGENT_ID, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID, $run.id)	
+
+	if ($thresholdViolationsCount -gt 0)
+	{
+		$thresholdMessage=("{0} thresholds violated." -f $thresholdViolationsCount)
+		$thresholdImage="bowtie-status-error"
+	}
+	else
+	{
+		$thresholdMessage="No thresholds violated."
+		$thresholdImage="bowtie-status-success"
+	}
+	$summary = ('[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUrl ,$run.name)
+	$summary = ('<span class="bowtie-icon {3}" />   {4}<br/>[Test Run: {0}]({1}) using {2}.<br/>' -f  $run.runNumber, $webResultsUri , $run.name, $thresholdImage, $thresholdMessage)
 
 	('<p>{0}</p>' -f $summary) >>  $summaryFile
```

Comment:
>('<p>{0}</p>' -f $summary) >>  $summaryFile [](start = 1, length = 43)

Anjani had changed this code to write in the ascii format. Am I missing something?